# translation of python-fedora.devel.es.po to Spanish
# Spanish translations for python-fedora.
# Copyright (C) 2009 ORGANIZATION
# This file is distributed under the same license as the python-fedora
# project.
#
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
# Gladys Guerrero Lozano <gguerrer@redhat.com>, 2009.
msgid ""
msgstr ""
"Project-Id-Version: python-fedora.devel.es\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2010-03-13 00:59-0500\n"
"PO-Revision-Date: 2010-03-18 13:26-0300\n"
"Last-Translator: Domingo Becker <domingobecker@gmail.com>\n"
"Language-Team: Spanish <trans-es@lists.fedoraproject.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"Generated-By: Babel 0.9.4\n"
"X-Poedit-Language: Spanish\n"

#: fedora/release.py:9
msgid "Python modules for interacting with Fedora Services"
msgstr "Módulos de Python para interactuar con los Servicios de Fedora"

#: fedora/release.py:10
msgid ""
"\n"
"The Fedora Project runs many different services.  These services help us to\n"
"package software, develop new programs, and generally put together the distro.\n"
"This package contains software that helps us do that.\n"
msgstr ""
"\n"
"El Proyecto Fedora ejecuta diferentes servicios. Estos servicios nos ayudan\n"
"a empaquetar software, desarrollar programas y por lo general, agruparlos en una distribución.\n"
"Este paquete contiene software que nos ayuda a hacer eso.\n"

#: fedora/textutils.py:79
#, python-format
msgid "to_unicode was given \"%(obj)s\" which is neither a byte string (str) or a unicode string"
msgstr "a to_unicode se le dió \"%(obj)s\" que no es una cadena de bytes (str) ni una cadena unicode"

#: fedora/textutils.py:82
#: fedora/textutils.py:140
#, python-format
msgid "non_string value, %(param)s, is not set to a valid action"
msgstr "el valor no cadena, %(param)s, no fue puesto para una acción válida"

#: fedora/textutils.py:137
#, python-format
msgid "to_bytes was given \"%(obj)s\" which is neither a unicode string or a byte string (str)"
msgstr "a to_bytes se le dió \"%(obj)s\" que no es una cadena de bytes (str) ni una cadena unicode"

#: fedora/accounts/fas2.py:8
msgid "fedora.accounts.fas2 has moved to fedora.client.fas2.  This location will disappear in 0.4"
msgstr "fedora.accounts.fas2 ha sido trasladado a fedora.client.fas2. Esta dirección desaparecerá en 0.4"

#: fedora/client/baseclient.py:85
#: fedora/client/baseclient.py:204
#: fedora/client/baseclient.py:223
#: fedora/client/baseclient.py:235
msgid "session_cookie is deprecated, use session_id instead"
msgstr "session_cookie está depreciado, use session_id en su lugar"

#: fedora/client/baseclient.py:103
#, python-format
msgid "Unable to load session from %(file)s"
msgstr "No se pudo cargar la sesión desde %(file)s"

#: fedora/client/baseclient.py:120
#, python-format
msgid "Unable to create %(dir)s: %(error)s"
msgstr "No se pudo crear %(dir)s: %(error)s"

#: fedora/client/baseclient.py:132
#, python-format
msgid "Unable to write to session file %(session)s: %(error)s"
msgstr "No se pudo escribir el archivo de sesión %(session)s: %(error)s"

#: fedora/client/baseclient.py:154
#, python-format
msgid "No session cached for \"%s\""
msgstr "No hay cache de sesión para \"%s\""

#: fedora/client/baseclient.py:276
#, python-format
msgid "send_request() got an unexpected keyword argument \"%s\""
msgstr "send_request() recibió un argumento \"%s\" inesperado"

#: fedora/client/baseclient.py:280
msgid "send_request() got an unexpected keyword argument \"input\""
msgstr "send_request() recibió un argumento de \"salida\" inesperado"

#: fedora/client/baseclient.py:284
msgid "send_request() got an unexpected keyword argument"
msgstr "send_request() recibió un argumento de palabra clave inesperado"

#: fedora/client/baseclient.py:288
msgid "send_request(input) is deprecated.  Use send_request(req_params) instead"
msgstr "send_request(input) está depreciado. Use send_request(req_params) en su lugar"

#: fedora/client/baseclient.py:304
msgid "Auth was requested but no way to perform auth was given.  Please set username and password or session_id before calling this function with auth=True"
msgstr "Autorización fue solicitada pero no hay manera de realizar el tipo dado. Por favor establezca el nombre de usuario, contraseña o identificador de sesión antes de llamar esta función con auth=True"

#: fedora/client/bodhi.py:191
msgid "You must specify a username"
msgstr "Debe especificar un nombre de usuario"

#: fedora/client/bodhi.py:260
#, python-format
msgid "Reading from %s "
msgstr "Leyendo desde %s"

#: fedora/client/fas2.py:218
#, python-format
msgid "FAS server unable to retrieve group %s"
msgstr "El servidor FAS no pudo obtener el grupo %s"

#: fedora/client/fas2.py:342
msgid "key must be one of \"id\", \"username\", or \"email\""
msgstr "la llave debe ser de \"id\", \"username\" o \"email\""

#: fedora/client/fas2.py:348
#, python-format
msgid "%(field)s is not a valid field to filter"
msgstr "%(field)s no es un campo válido para filtrar"

#: fedora/client/fas2.py:397
msgid "people_by_id() is deprecated and will be removed in 0.4.  Please port your code to use people_by_key(key='id', fields=['human_name', 'email', 'username', 'bugzilla_email']) instead"
msgstr "people_by_id() está depreciado y será eliminado en 0.4. Por favor porte su código para usar people_by_key(key='id', fields=['human_name', 'email', 'username', 'bugzilla_email']) en su lugar"

#: fedora/client/fas2.py:579
msgid "FAS server unable to retrieve group members"
msgstr "El servidor FAS no pudo obtener los miembros del grupo"

#: fedora/client/fas2.py:601
msgid "FAS server unable to retrieve user information"
msgstr "El servidor FAS no pudo recuperar la información del usuario"

#: fedora/client/fasproxy.py:67
msgid "FasProxyClient() got an unexpected keyword argument 'session_as_cookie'"
msgstr "FasProxyClient() obtuvo un argumento palabra clave inesperado 'session_as_cookie'"

#: fedora/client/pkgdb.py:182
#, python-format
msgid "We do not have enough information to create package %(pkg)s. Need version owner."
msgstr "No tenemos suficiente información para crear paquete  %(pkg)s. Se necesita propietario de versión."

#: fedora/client/pkgdb.py:193
#, python-format
msgid "PackageDB returned an error creating %(pkg)s: %(msg)s"
msgstr "PackageDB regresó un error al crear %(pkg)s: %(msg)s"

#: fedora/client/pkgdb.py:216
#: fedora/client/pkgdb.py:272
#, python-format
msgid "Unable to save all information for %(pkg)s: %(msg)s"
msgstr "No se pudo guardar toda la información para los paquetes %(pkg)s: %(msg)s"

#: fedora/client/pkgdb.py:307
#, python-format
msgid "Collection abbreviation %(collection)s is unknown.  Use F, FC, EL, or OLPC"
msgstr "Abreviatura de colección %(collection)s es desconocida. Use F, FC, EL o OLPC"

#: fedora/client/pkgdb.py:332
msgid "collection and collection_ver are deprecated names for collctn_name and collctn_ver respectively.  Ignoring the values given in them."
msgstr "collection y collection_ver son nombres que ya no se usan, ahora usar collctn_name y collctn_ver respectivamente.  Ignorando los valores dados en ellos."

#: fedora/client/pkgdb.py:338
msgid ""
"collection has been renamed to collctn_name.\n"
"Please start using the new name.  collection will go away in 0.4.x."
msgstr ""
"collection fue renombrado a collctn_name.\n"
"Por favor, comience a usar el nombre nuevo.  collection será eliminado en 0.4.x."

#: fedora/client/pkgdb.py:344
msgid ""
"collection_ver has been renamed to collctn_ver.\n"
"Please start using the new name.  collection_ver will go away in 0.4.x."
msgstr ""
"collection_ver fue renombrado a collctn_ver.\n"
"Por favor, comience a usar el nombre nuevo.  collection_ver será eliminado en 0.4.x."

#: fedora/client/pkgdb.py:381
msgid ""
"collectn_list is a deprecated name for collctn_list.\n"
"Ignoring the value of collectn_list."
msgstr ""
"collectn_list ya no se usa, ahora usar collctn_list.\n"
"Ignorando el valor de collectn_list."

#: fedora/client/pkgdb.py:386
msgid ""
"collectn_list has been renamed to collctn_list.\n"
"Please start using the new name.  collectn_list will go away in 0.4.x."
msgstr ""
"collectn_list fue renombrado a collctn_list.\n"
"Por favor, comience a usar el nombre nuevo.  collectn_list será eliminado en 0.4.x."

#: fedora/client/pkgdb.py:460
msgid ""
"collectn is a deprecated name for collctn.\n"
"Ignoring the value of collectn."
msgstr ""
"collectn fue renombrado a collctn.\n"
"Ignorando el valor de collectn."

#: fedora/client/pkgdb.py:465
msgid ""
"collectn has been renamed to collctn.\n"
"Please start using the new name.  collectn will go away in 0.4.x."
msgstr ""
"collectn fue renombrado a collctn.\n"
"Por favor, comience a usar el nombre nuevo.  collectn será eliminado en 0.4.x."

#: fedora/client/pkgdb.py:475
#, python-format
msgid "Collection shortname %(collctn)s is unknown."
msgstr "Abreviatura de colección %(collctn)s es desconocida."

#: fedora/client/proxyclient.py:149
msgid "proxyclient.__init__:entered"
msgstr "proxyclient.__init__: entrante"

#: fedora/client/proxyclient.py:158
msgid "Returning cookies from send_request() is deprecated and will be removed in 0.4.  Please port your code to use a session_id instead by calling the ProxyClient constructor with session_as_cookie=False"
msgstr "Regresar cookies desde send_request está depreciado y será eliminado en 0.4. Por favor porte su código para usar session_id en su lugar llamando al constructor ProxyClient con session_as_cookie=False"

#: fedora/client/proxyclient.py:164
msgid "proxyclient.__init__:exited"
msgstr "proxyclient.__init__: saliente"

#: fedora/client/proxyclient.py:236
msgid "proxyclient.send_request: entered"
msgstr "proxyclient.send_request: entrante"

#: fedora/client/proxyclient.py:245
msgid "Giving a cookie to send_request() to authenticate is deprecated and will be removed in 0.4. Please port your code to use session_id instead."
msgstr "Ofrecer una cookie a send_request() para autenticarse está depreciado y será eliminado en 0.4. Por favor porte su código para usar session_id en su lugar."

#: fedora/client/proxyclient.py:255
msgid "username and password must both be set in auth_params"
msgstr "nombre de usuario y contraseña deben estar establecidos en auth_params"

#: fedora/client/proxyclient.py:258
msgid "No known authentication methods specified: set \"cookie\" in auth_params or set both username and password in auth_params"
msgstr "No se especificaron métodos de autenticación conocidos: establezca \"cookie\" en auth_params o establezca nombre de usuario y contraseña en auth_params"

#: fedora/client/proxyclient.py:315
#, python-format
msgid "Creating request %(url)s"
msgstr "Creando petición %(url)s"

#: fedora/client/proxyclient.py:316
#, python-format
msgid "Headers: %(header)s"
msgstr "Encabezados: %(header)s"

#: fedora/client/proxyclient.py:320
#, python-format
msgid "Data: %(data)s"
msgstr "Datos: %(data)s"

#: fedora/client/proxyclient.py:332
msgid "Authentication failed logging in"
msgstr "Autenticación falló en el inicio"

#: fedora/client/proxyclient.py:334
msgid "Unable to log into server.  Invalid authentication tokens.  Send new username and password"
msgstr "No pudo iniciar en el servidor. Credenciales de autenticación inválidas. Envíe un nombre de usuario y contraseña"

#: fedora/client/proxyclient.py:340
msgid "Unknown HTTP Server Response"
msgstr "Respuesta de Servidor HTTP Desconocida"

#: fedora/client/proxyclient.py:362
#, python-format
msgid "Error returned from simplejson while processing %(url)s: %(err)s"
msgstr "Error de simplejson mientras se procesaba %(url)s: %(err)s"

#: fedora/client/proxyclient.py:377
msgid "proxyclient.send_request: exited"
msgstr "proxyclient.send_request: saliente"

#: fedora/client/wiki.py:64
#, python-format
msgid "Login failed: %s"
msgstr "Entrada fallida: %s"

#: fedora/client/wiki.py:85
#, python-format
msgid "From %(then)s to %(now)s"
msgstr "Desde %(then)s hasta %(now)s"

#: fedora/client/wiki.py:88
#, python-format
msgid "%d wiki changes in the past week"
msgstr "%d cambios en la wiki en la semana pasada"

#: fedora/client/wiki.py:90
msgid ""
"Warning: Number of changes reaches the API return limit.\n"
"You will not get the complete list of changes unless\n"
"you run this script using a 'bot' account."
msgstr ""
"Advertencia: El número de cambios ha alcanzado el límite de regreso de la API.\n"
"Usted no tendrá la lista completa de cambios a menos que\n"
"ejecute este programa usando una cuenta 'bot'."

#: fedora/client/wiki.py:100
msgid ""
"\n"
"== Most active wiki users =="
msgstr ""
"\n"
"== Usuarios wiki más activos =="

#: fedora/client/wiki.py:107
msgid ""
"\n"
"== Most edited pages =="
msgstr ""
"\n"
"== Páginas más editadas =="

#: fedora/compat25/defaultdict.py:40
msgid "First argument must be callable"
msgstr "El primer argumento debe poderse llamar"

#: fedora/django/auth/models.py:50
msgid "Loading FAS groups..."
msgstr "Cargando grupos FAS..."

#: fedora/django/auth/models.py:56
msgid "Unable to load FAS groups. Did you set FAS_USERNAME and FAS_PASSWORD?"
msgstr "No se pudieron cargar los grupos FAS. ¿Se establecieron FAS_USERNAME y FAS_PASSWORD?"

#: fedora/django/auth/models.py:63
msgid "FAS groups loaded. Don't forget to unset FAS_USERNAME and FAS_PASSWORD."
msgstr "Grupos de FAS cargados. No olvide eliminar FAS_USERNAME y FAS_PASSWORD."

#: fedora/tg/client.py:8
msgid "fedora.tg.client has moved to fedora.client.  This location will disappear in 0.4"
msgstr "fedora.tg.client se ha trasladado a fedora.client. Esta dirección desaparecerá en 0.4"

#: fedora/tg/controllers.py:60
#, python-format
msgid "Welcome, %s"
msgstr "Bienvenido, %s"

#: fedora/tg/controllers.py:69
msgid "The credentials you supplied were not correct or did not grant access to this resource."
msgstr "Las credenciales que se proporcionaron no son correctas o no proveen el acceso a este recurso."

#: fedora/tg/controllers.py:72
msgid "You must provide your credentials before accessing this resource."
msgstr "Debe proveer sus credenciales antes de acceder a este recurso."

#: fedora/tg/controllers.py:75
msgid "Please log in."
msgstr "Por favor, inicie sesión."

#: fedora/tg/controllers.py:100
msgid "You have successfully logged out."
msgstr "Ha cerrado con éxito su sesión."

#: fedora/tg/tg1utils.py:100
msgid "url() expects a dictionary for query parameters"
msgstr "url() espera un diccionario como parámetros de búsqueda"

#: fedora/tg/util.py:40
msgid "fedora.tg.util is deprecated.  Switch to one of these instead:   TG1 apps: fedora.tg.tg1utils   TG2 apps: fedora.tg.tg2utils.  This file will disappear in 0.4"
msgstr "fedora.tg.util ya no se usa.  Cambie a uno de estos:   TG1 apps: fedora.tg.tg1utils   TG2 apps: fedora.tg.tg2utils.  Este archivo desaparecerá en 0.4"

#: fedora/tg/identity/jsonfasprovider1.py:227
#: fedora/tg/identity/jsonfasprovider2.py:389
#, python-format
msgid "Error logging in %(user)s: %(error)s"
msgstr "Error al iniciar sesión %(user)s: %(error)s"

#: fedora/tg/identity/jsonfasprovider2.py:60
msgid "Cannot enable ssl certificate auth via identity.ssl without setting fas.usernamme and fas.password for authorization"
msgstr "No se puede habilitar el certificado ssl vía identity.ssl sin configurar fas.username y fas.password para autorización"

#: fedora/tg/identity/jsonfasprovider2.py:155
#, python-format
msgid "jsonfasprovider, ssl, returned errors from send_request: %s"
msgstr "jsonfasprovider, ssl, regresó errores de send_request: %s"

#: fedora/tg/identity/jsonfasprovider2.py:166
#, python-format
msgid "jsonfasprovider returned errors from send_request: %s"
msgstr "jsonfasprovider regresó errores de send_request: %s"

#: fedora/tg/templates/genshi/login.html:10
msgid "Log In"
msgstr "Iniciar sesión"

#: fedora/tg/templates/genshi/login.html:14
msgid "User Name:"
msgstr "Nombre de Usuario:"

#: fedora/tg/templates/genshi/login.html:17
msgid "Password:"
msgstr "Contraseña:"

#: fedora/tg/templates/genshi/login.html:21
#: fedora/tg/templates/genshi/login.html:84
msgid "Login"
msgstr "Entrar"

#: fedora/tg/templates/genshi/login.html:28
msgid "CSRF attacks"
msgstr "ataques CSRF"

#: fedora/tg/templates/genshi/login.html:29
msgid ""
" are a means for a malicious website to make a request of another\n"
"        web server as the user who contacted the malicious web site.  The\n"
"        purpose of this page is to help protect your account and this server\n"
"        from attacks from such malicious web sites.  By clicking below, you are\n"
"        proving that you are a person rather than just the web browser\n"
"        forwarding your authentication cookies on behalf of a malicious\n"
"        website."
msgstr ""
" son un medio para que un sitio web maligno haga un pedido de otro\n"
"        sitio web como el usuario que contactó el sitio maligno. El\n"
"        propósito de este paquete es ayudar a proteger su cuenta y a este servidor\n"
"        de ataques de dichos sitios. Haciendo clic abajo, estará probando\n"
"        que Ud. es la persona en vez del navegador de web que usó sus cookies de\n"
"        autenticación como si fuera el sitio\n"
"        web maligno."

#: fedora/tg/templates/genshi/login.html:36
msgid "I am a human"
msgstr "Soy un humano"

#: fedora/tg/templates/genshi/login.html:41
msgid "Forgot Password?"
msgstr "¿Olvidó la contraseña?"

#: fedora/tg/templates/genshi/login.html:42
msgid "Sign Up"
msgstr "Registrarse"

#: fedora/tg/templates/genshi/login.html:60
msgid "Welcome"
msgstr "Bienvenido"

#: fedora/tg/templates/genshi/login.html:82
msgid "You are not logged in"
msgstr "No ha ingresado al sistema"

#: fedora/tg/templates/genshi/login.html:89
msgid "CSRF protected"
msgstr "Protegido por CSRF"

#: fedora/tg/templates/genshi/login.html:91
msgid "Verify Login"
msgstr "Verificar Entrada"

#: fedora/tg/templates/genshi/login.html:97
msgid "Logout"
msgstr "Salir"

#: fedora/tg/visit/jsonfasvisit1.py:107
#: fedora/tg/visit/jsonfasvisit2.py:108
#, python-format
msgid "updating visit (%s)"
msgstr "actualizando visita (%s)"

#: fedora/wsgi/csrf.py:117
msgid "Creating CSRFProtectionMiddleware"
msgstr "Creando CSRFProtectionMiddleware"

#: fedora/wsgi/csrf.py:126
#, python-format
msgid "clean_environ(%s)"
msgstr "clean_environ(%s)"

#: fedora/wsgi/csrf.py:129
#, python-format
msgid "Deleting %(key)s from environ"
msgstr "Borrando %(key)s del entorno"

#: fedora/wsgi/csrf.py:141
#, python-format
msgid "CSRFProtectionMiddleware(%(r_path)s)"
msgstr "CSRFProtectionMiddleware(%(r_path)s)"

#: fedora/wsgi/csrf.py:148
msgid "User supplied CSRF token matches environ!"
msgstr "¡El token CSRF del usuario coincide con el entorno!"

#: fedora/wsgi/csrf.py:151
msgid "Clearing identity"
msgstr "Limpiando identidad"

#: fedora/wsgi/csrf.py:154
#, python-format
msgid "Invalid CSRF token.  User supplied (%(u_token)s) does not match what's in our environ (%(e_token)s)"
msgstr "Token CSRF inválido.  El (%(u_token)s) dado por el usuario no coincide con el de nuestro entorno (%(e_token)s)"

#: fedora/wsgi/csrf.py:162
msgid "CSRF_AUTH_STATE; rewriting headers"
msgstr "CSRF_AUTH_STATE; rescribiendo cabeceras"

#: fedora/wsgi/csrf.py:168
#, python-format
msgid "response.location = %(r_loc)s"
msgstr "response.location = %(r_loc)s"

#: fedora/wsgi/csrf.py:240
#, python-format
msgid "CSRFMetadataProvider.add_metadata(%(r_path)s)"
msgstr "CSRFMetadataProvider.add_metadata(%(r_path)s)"

#: fedora/wsgi/csrf.py:246
#, python-format
msgid "session_id = %(s_id)r"
msgstr "session_id = %(s_id)r"

#: fedora/wsgi/csrf.py:251
msgid "Identity updated with CSRF token"
msgstr "Identidad actualizada con el token CSRF"

#: fedora/wsgi/csrf.py:254
msgid "Setting CSRF_AUTH_STATE"
msgstr "Poniendo el CSRF_AUTH_STATE"

#: fedora/wsgi/csrf.py:264
msgid "Got HTTPFound(302) from repoze.who.application"
msgstr "Se obtuvo HTTPFound(302) de repoze.who.application"

#: fedora/wsgi/csrf.py:270
#, python-format
msgid "Altered headers: %(headers)s"
msgstr "Cabeceras alteradas: %(headers)s"

#: fedora/wsgi/csrf.py:273
#, python-format
msgid "Invalid session cookie %(s_id)r, not setting CSRF token!"
msgstr "Cookie de sesión inválido %(s_id)r, ¡no se pone el token CSRF!"

#: fedora/wsgi/csrf.py:283
#, python-format
msgid "%(token)s in GET"
msgstr "%(token)s en GET"

#: fedora/wsgi/csrf.py:290
#, python-format
msgid "%(token)s in POST"
msgstr "%(token)s en POST"

#: fedora/wsgi/faswho/faswhoplugin.py:110
msgid "Keep alive cache miss"
msgstr "Se perdió el keep alive del caché"

#: fedora/wsgi/faswho/faswhoplugin.py:124
msgid "in identify()"
msgstr "en identify()"

#: fedora/wsgi/faswho/faswhoplugin.py:131
#, python-format
msgid "Request identify for cookie %(cookie)s"
msgstr "Se pide identificación para el cookie %(cookie)s"

#: fedora/wsgi/faswho/faswhoplugin.py:155
msgid "In remember()"
msgstr "En remember()"

#: fedora/wsgi/faswho/faswhoplugin.py:168
msgid "In forget()"
msgstr "En forget()"

#: fedora/wsgi/faswho/faswhoplugin.py:175
#, python-format
msgid "Forgetting login data for cookie %(s_id)s"
msgstr "Desechando datos de ingreso para el cookie %(s_id)s"

#: fedora/wsgi/faswho/faswhoplugin.py:192
msgid "In authenticate()"
msgstr "En authenticate()"

#: fedora/wsgi/faswho/faswhoplugin.py:216
msgid "Authentication failed, setting error"
msgstr "La autenticación falló, poniendo el error"

#: fedora/wsgi/faswho/faswhoplugin.py:237
msgid "An unknown error happened when trying to log you in.  Please try again."
msgstr "Un error desconocido ocurrió cuando Ud. intentó ingresar.  Por favor, intente nuevamente."

#: fedora/wsgi/faswho/faswhoplugin.py:248
msgid "Metadata cache miss - refreshing metadata"
msgstr "Falta el cache de metadatos - refrescando los metadatos"

#: fedora/wsgi/faswho/faswhoplugin.py:279
msgid "In add_metadata"
msgstr "En add_metadata"

#: fedora/wsgi/faswho/faswhoplugin.py:283
msgid "Error exists in session, no need to set metadata"
msgstr "Hay error en la sesión, no hace falta poner los metadatos"

#: fedora/wsgi/faswho/faswhoplugin.py:293
#, python-format
msgid "Request metadata for cookie %(cookie)s"
msgstr "Se solicita metadatos para el cookie %(cookie)s"

#: fedora/wsgi/test/controllers/root.py:31
msgid "Wrong credentials"
msgstr "Credenciales incorrectas"

#: fedora/wsgi/test/controllers/root.py:46
#, python-format
msgid "Welcome back, %s!"
msgstr "¡Bienvenido, %s!"

#: fedora/wsgi/test/controllers/root.py:56
msgid "We hope to see you soon!"
msgstr "¡Esperamos verlo pronto!"

